src/pages/api/nft/collectors/[chain]/[address].ts

```diff
@@ -32,6 +32,16 @@ export const alchemyApiUrl: Record<ContractCallSupportedChain, string> = {
   ZKSYNC_ERA: "",
   LINEA: "",
   CYBER: "",
+  ARBITRUM: "",
```

Comment:
Note that in guild-gate we are using `https://arb-mainnet.g.alchemy.com/nft/v2/{apikey}`, so we might be able to add an alchemy link here, but for that I will need help @BrickheadJohnny

Reply:
Oh I see, we usually separate projects (core - gate - frontend) for some reason on Alchemy, but actually the frontend won't call these endpoints frequently, so I think it's okay to add it here. I implemented the necessary change in https://github.com/guildxyz/guild.xyz/pull/1298/commits/ea0053a9f31d58985e3475e5cf54b854d409994c - we'll just need to add the necessary environment variable on Vercel. 👌